packages/alt-ergo/alt-ergo.0.95.2/opam

```diff
@@ -1,24 +1,34 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
```

Comment:
It is generally good form to ping maintainers when making metadata updates. I'd rather not have all this noise in the metadata files but I also don't think it's something that causes an issue.

In the case of this package, I'm not clear on what was the initial problem that prompted this warning. From a public package user's perspective, https://github.com/OCamlPro/alt-ergo is the "development" or source control repo as the project is developed privately. Did the addition of this information interfere with something?